README.md

```diff
@@ -3,48 +3,37 @@
 API plugins, microservices, and commands for accessing NetCDF data/metadata contained in iRODS data objects.
 
 Currently supported version(s) of iRODS:
-   - 4.2.8
-
-OS Distributions presently supported:
-   - Centos 7
-   - Ubuntu 16.04
-   - Ubuntu 18.04
+   - 4.3.1
+   - 4.2.12
 
 Automated building and packaging is available as before, but is now done through CMake.
 
-## Building the Plugin for iRODS 4.2+
+## Building the NetCDF Plugin
 
-Make sure the development packages for OpenSSL, NetCDF, HDF5, and iRODS itself are installed on the build machine, as well
-as these build externals:
+Dependencies for building and running the plugin are as follows:
 
-   - `irods-externals-cmake3.11.4-0`
-   - `irods-externals-clang6.0-0`
-   - `irods-externals-boost1.67.0-0`
+   - irods-externals-cmake
+   - irods-externals-clang
+   - irods-externals-clang-runtime
+   - irods-externals-boost
 
-Clone this repository:
-```sh
-git clone -b {BRANCH|TAG|COMMIT} http:/github.com/irods/irods_netcdf
-```
-(Currently  `4.2.8.0` is the only plugin version available on iRODS 4.2+, and you must be running the 4.2.8 release of the server.)
+Also make sure the development packages for OpenSSL, NetCDF, and HDF5, as well as iRODS itself, are installed on the build machine.
+
+After cloning the `irods_netcdf` repository locally, perform these steps to build:
 
-3. Build:
 ```sh
 mkdir build ; cd build
-/opt/irods-externals/cmake3.11.4-0/bin/cmake ../irods_netcdf
-make -j7
-```
-Then:
-```
-make
+cmake <irods_netcdf_repository_directory>
+make -j
 ```
 
 ---
 
-## Packages
+## Building Packages
 
 Using the build command
 ```
-make package
+make [-j] package
```

Comment:
There are two references to `make -j`. One here and one on line 27.

Seems these two sections, **Building Packages** and **Building the NetCDF Plugin**, cover the same material.

Consider consolidating them.

Reply:
Consolidated...